clerk/sessions_test.go

```diff
@@ -252,3 +294,10 @@ const dummySessionJson = `{
         "status": "ended",
         "user_id": "user_1mebQggrD3xO5JfuHk7clQ94ysA"
     }`
+
+const dummySessionTokenJsonTemplate = `
```

Comment:
```suggestion
const dummySessionTokenJSONTemplate = `
```

Looks like this constant is used only in one place. We can add it inside the test where it's actually used.